ios/Classes/Blemulator/Bridging/DartMethodCaller.m

```diff
@@ -108,6 +109,33 @@ - (void)isDeviceConnected:(NSString *)deviceIdentifier
                                                                           onError:errorHandler]];
 }
 
+// MARK: - Discovery
+
+- (void)discoverAllServicesAndCharacteristics:(NSString *)deviceIdentifier
+                                         name:(NSString *)name
+                                transactionId:(NSString *)transactionId
+                                      resolve:(Resolve)resolve
+                                       reject:(Reject)reject {
+    NSDictionary<NSString *,id> *arguments = [NSDictionary dictionaryWithObjectsAndKeys:
+                                              deviceIdentifier, DART_CALL_ARGUMENT_DEVICE_ID,
+                                              nil];
+    SuccessHandler successHandler = ^(id result) {
+        resolve([DartResultConverter deviceContainerFromDartResult:result
+                                                        peripheral:[[Peripheral alloc] initWithIdentifier:deviceIdentifier
+                                                                                                     name:name
+                                                                                                      mtu:23]]);
+    };
+    ErrorHandler errorHandler = ^(id error) {
+        // TODO: - Send error here
```

Comment:
Added error throwing capabilities. 

Since `invokeMethod:arguments:result` should result in either: `FlutterError`, `FlutterMethodNotImplemented` or `any other result as success` I've changed the `invokeMethodResultHandlerForMethod:onSuccess:onError` accordingly.

(reference: https://api.flutter.dev/objcdoc/Classes/FlutterMethodChannel.html#/c:objc(cs)FlutterMethodChannel(im)invokeMethod:arguments:result:)